PendingReleaseNotes

```diff
@@ -1,6 +1,8 @@
 v10.0.0
 ======
 
+* [FileStore] removed empty lfn_close function and calls to it.
+
```

Comment:
Ok, I remove it. I saw email about adding change description along commit so I thought this would be the place.